rcore/appmanager_app_runloop.c

```diff
@@ -36,6 +36,8 @@ void appmanager_post_generic_app_message(AppMessage *am, TickType_t timeout)
  */
 void appmanager_app_main_entry(void)
 {
+    /* This is cheesy, but there is a condition there the thread isn't assigned yet */
+    vTaskDelay(10);
```

Comment:
Is there a way to solve, rather than evading, this race condition?

Reply:
Yes, I do have it lined up to cover this hack